plugins/state_history_plugin/state_history_plugin_abi.cpp

```diff
@@ -95,15 +95,15 @@ extern const char* const state_history_plugin_abi = R"({
             "name": "action_trace_v0", "fields": [
                 { "name": "action_ordinal", "type": "varuint32" },
                 { "name": "creator_action_ordinal", "type": "varuint32" },
-                { "name": "parent_action_ordinal", "type": "varuint32" },
                 { "name": "receipt", "type": "action_receipt?" },
```

Comment:
What about `closest_unnotified_ancestor_action_ordinal` ?

Reply:
We decided not to include it in state_history_plugin since we don't want to encourage clients to deal with the ambiguous execution tree.